lib/model/folder_sendrecv.go

```diff
@@ -357,7 +357,12 @@ func (f *sendReceiveFolder) processNeeded(snap *db.Snapshot, dbUpdateChan chan<-
 				// files to delete inside them before we get to that point.
 				dirDeletions = append(dirDeletions, file)
 			} else if file.IsSymlink() {
-				f.deleteFile(file, snap, dbUpdateChan, scanChan)
+				if runtime.GOOS == "windows" {
+					// We never had it, just pretend we deleted it
+					dbUpdateChan <- dbUpdateJob{file, dbUpdateDeleteFile}
```

Comment:
This might not strictly be true, if it's a file that we synced once that then changed to be a symlink and then finally was deleted -- we would still have the file around? Or maybe that case would have been handled differently to begin with or picked up by the scanner after we ignored the symlink change...?

Reply:
Good point - I moved the fix into `deleteFile`.

Unrelated to the PR but a similar problem: We also don't fully handle valid file to symlink on windows: We do mark it as invalid and synced, but we do not check if there's an old file and remove that. Quite likely on next scan we then pick up the old file and give it a new version, overriding the symlink. Need to check that out.